packages/block-editor/src/components/link-control/README.md

```diff
@@ -70,34 +110,33 @@ Whether to present initial suggestions immediately.
 - Type: `boolean`
 - Required: No
 
-If passed as either `true` or `false`, controls the internal editing state of the component to respective show or not show the URL input field.
-
+If passed as either `true` or `false`, controls the internal editing state of the component to respectively show or not to show the URL input field.
```

Comment:
```suggestion
Controls the internal editing state of the component. If passed as either `true` or `false` will respectively show or hide the URL input field.
```
Rewording to make this clearer.

Also can we check whether it really has to be `true` or `false` as strict value types. For example if I pass `1` and it is cooerced into a boolean then does this cause some other unexpected behaviour? How about `undefined`?